airbyte-webapp/src/core/resources/DestinationDefinition.ts

```diff
@@ -16,6 +16,7 @@ export default class DestinationDefinitionResource
   readonly latestDockerImageTag: string = "";
   readonly documentationUrl: string = "";
   readonly icon: string = "";
+  readonly releaseStage?: releaseStage = undefined;
```

Comment:
:art: code style. `undefined` is anyway the default for a variable, so we don't need to explicitly assign it.

```suggestion
  readonly releaseStage?: releaseStage;
```